docs/mibi/v1.yaml

```diff
@@ -1,5 +1,8 @@
 # Generated YAML: PRs should not start here!
```

Comment:
Please note the protocol for submitting PRs.

Reply:
I'll take your changes and implement them, but anything you can do to follow the protocol would help me.